nx-X11/programs/Xserver/Xext/xf86bigfont.c

```diff
@@ -638,12 +638,12 @@ ProcXF86BigfontQueryFont(
 	        + (nCharInfos+1)/2 * 2 * sizeof(CARD16)
 	      : 0);
 	xXF86BigfontQueryFontReply* reply =
-	   (xXF86BigfontQueryFontReply *) ALLOCATE_LOCAL(rlength);
+	   (xXF86BigfontQueryFontReply *) calloc(1, rlength);
```

Comment:
I had added a missing memset here. When replacing ALLOCATE_LOCAL I noticed I can remove the memset again and just use calloc.

calloc is better, right, but keep in mind that a calloc is slower, also.